en/app.ftl

```diff
@@ -219,30 +262,53 @@ landing-faq-cta = See more FAQs about { -brand-name-firefox-relay }
 ## Premium promo page
 
 premium-promo-hero-headline = Make protecting your inbox easier with { -brand-name-firefox-relay-premium }
+# Deprecated
+premium-promo-hero-body-html = With { -brand-name-firefox-relay-premium }, you get unlimited custom email aliases that forward only the emails you want to your real email address. <b>For a limited time, you can upgrade to Relay Premium for only { $monthly_price } a month.</b>
```

Comment:
should we remove this string as there is no replacement string for it?

Reply:
There is `premium-promo-hero-body-html-2`, though I now realise that it should end in `-html`, so I changed that to `premium-promo-hero-body-2-html` in 2da7a82ae1ace69792eaed94270ca7d8407bba7e.